routes/web.php

```diff
@@ -310,6 +310,30 @@
 
     });
 
+    /*
+     * Routes related to leaderboards.
+     */
+    Route::group(['prefix' => 'leaderboards', 'as' => 'leaderboards::'], function () {
+        Route::get('', ['as' => 'index', 'uses' => 'LeaderboardController@index']);
+        Route::get('list', ['as' => 'admin', 'middleware' => ['auth', 'permission:board'], 'uses' => 'LeaderboardController@adminIndex']);
+
+        Route::get('add', ['as' => 'add', 'middleware' => ['auth', 'permission:board'], 'uses' => 'LeaderboardController@create']);
+        Route::post('add', ['as' => 'add', 'middleware' => ['auth', 'permission:board'], 'uses' => 'LeaderboardController@store']);
+
+        Route::get('edit/{id}', ['as' => 'edit', 'middleware' => ['permission:board'], 'uses' => 'LeaderboardController@edit']);
+        Route::post('edit/{id}', ['as' => 'edit', 'middleware' => ['permission:board'], 'uses' => 'LeaderboardController@update']);
+        Route::get('delete/{id}', ['as' => 'delete', 'middleware' => ['permission:board'], 'uses' => 'LeaderboardController@destroy']);
+    });
+
+    /*
+    * Routes related to leaderboard entries.
+    */
+    Route::group(['prefix' => 'leaderboards/entries', 'as' => 'leaderboards_entries::'], function () {
+        Route::get('', ['as' => 'index', 'uses' => 'LeaderboardEntryController@index']);
```

Comment:
I don't think it is necessary to index all the leaderboard entries separately from the leaderboards they're associated to

Reply:
I'm pretty sure this route is now obsolete, so I'll remove it in an upcoming commit.